frontend/src/features/projects/erp/components/ApprovalActions.tsx

```diff
@@ -88,7 +97,17 @@ export const ApprovalActions = ({
                 setCancel(false);
               }}
               title="Really Cancel Project?"
-              message={cancellationWarning}
+              message={
```

Comment:
The review page was showing the subdivision warning correctly, but not the erp/spl pages. This corrects that.